<!--
  Copyright (C) 2018-2020 LEIDOS.
  Licensed under the Apache License, Version 2.0 (the "License"); you may not
  use this file except in compliance with the License. You may obtain a copy of
  the License at
  http://www.apache.org/licenses/LICENSE-2.0
  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
  License for the specific language governing permissions and limitations under
  the License.
-->
<!--
  This file is used to launch the carla_ackermann_control_wrapper_node 
-->
<launch>
    <!-- Handle global parameters for variables-->
    <remap from="~vehicle_wheel_base" to="/vehicle_wheel_base"/> 
    <remap from="~vehicle_acceleration_limit" to="/vehicle_acceleration_limit"/>
    <remap from="~vehicle_deceleration_limit" to="/vehicle_deceleration_limit"/>
    <remap from="~vehicle_steer_lim_deg" to="/vehicle_steer_lim_deg"/> 

    <!--carla_ackermann_control_wrapper node-->
    <node name="carla_ackermann_control_wrapper" pkg="carla_ackermann_control_wrapper" type="carla_ackermann_control_wrapper_node">
        <rosparam command="load" file="$(find carla_ackermann_control_wrapper)/config/parameters.yaml"/>
    </node>
</launch>